src/data/js/5_clickHandler.js

```diff
@@ -8387,6 +8383,55 @@ function getSelector(host) {
         '.modal[style*="block"] .modal-footer input',
         '.modal[style*="block"] button[id*="AcceptButton"]'
       );
+    case "bulkapothecary.com":
+      return 'button.termly-styles-module-root-f61419:nth-child(1)';
+    case "cwauthors.com":
+      return 'button.cookies_btn';
+    case "deep-shine.de":
+      return 'button.btn:nth-child(4)';
+    case "dirsyncpro.org":
+      return '.cookie-notice-close';
+    case "druhotnesuroviny.cz":
+      return 'a.js-cc-banner-confirmAll-btn:nth-child(2)';
+    case "gera.de":
+      return 'button.text-nowrap:nth-child(1)';
+    case "jutarnji.hr":
+      return '#didomi-notice-agree-button';
+    case "lesciseauxmagiques.fr":
+      return '#soy-gtm-disallow-cookies';
+    case "mazda-autohaus-kaschmieder-waren.de":
+      return 'button._1e1wg:nth-child(2)';
+    case "monkeytype.com":
+      return '.rejectAll';
+    case "namesilo.com":
+      return 'button.bgcw';
+    case "vantilburgonline.nl":
+      return 'button.btn:nth-child(1)';
+    case "thunhoon.com":
+      return '.privacy-concern-close-button';
+    case "sparkasse-karlsruhe.de":
+      return '.secondary';
+    case "rugbycoachweekly.net":
+      return '#aos-Cookie-Modal-Accept';
+    case "rockhard.de":
+      return '.sp_choice_type_11';
+    case "prenatal.nl":
+      return 'button.btn:nth-child(3)';
+    case "plus.nl":
+      return '.gtm-cookies-popup-accept-all-btn';
+    case "neckermann.at":
+      return '.cookieSelectOverlayInlineSave > a:nth-child(2)';
+    case "bernard.fr":
+      return 'button.ot-btn:nth-child(2)';
+    case "as.com":
+      return '._5r2nlf4';
```

Comment:
Shouldn't be a randomly generated ID, use `#redesignCmpWrapper a[href="https://as.com/"]` as it should remain static.